lib/auth/keystore/keystore_test.go

```diff
@@ -349,6 +371,53 @@ func TestKeyStore(t *testing.T) {
 			}
 		})
 	}
+
+	for _, tc := range backends {
+		t.Run(tc.desc+"_DeleteUnusedKeys", func(t *testing.T) {
+			if tc.shouldSkip() {
+				t.SkipNow()
+				return
```

Comment:
The return here is probably not necessary:

> SkipNow marks the test as having been skipped and stops its execution by calling runtime.Goexit.

Reply:
right, removed